app/views/layouts/ats/_navbar.html.slim

```diff
@@ -1,7 +1,11 @@
-ruby:
-  is_ats_profile = request.fullpath.eql?("/ats/profile")
+- account_icon_dropdown_elements = capture do
+    li.nav-item class=("active" if request.fullpath.eql?("/ats/settings"))
+      = link_to t("core.settings"), ats_settings_path, class: "nav-link"
+    li.nav-item
+      = button_to t("rodauth.logout_button"), rodauth.logout_path, method: :post, class: "nav-link",
+                  form: { data: { turbo: false } }
```

Comment:
Why does turbo have to be turned off for sign out link?
It seems to work without this attribute too

Reply:
no idea why [it was added by Dmitry](https://github.com/toughbyte/ats/pull/98/files). let's remove